com.cburch.Logisim.yml

```diff
@@ -0,0 +1,45 @@
+app-id: com.cburch.Logisim
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: logisim
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
I misunderstood `--socket=wayland` purpose, my bad.